src/Aspire.Hosting.Garnet/GarnetContainerImageTags.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Garnet;
+
+internal static class GarnetContainerImageTags
+{
+    public const string Registry = "ghcr.io";
+    public const string Image = "microsoft/garnet";
+    public const string Tag = "sha-d4c89fd";
```

Comment:
This shouldn't be a hard-coded version. We would need to constantly update our library every time a new version of Garnet is released.

Unfortunately looking at https://github.com/microsoft/garnet/pkgs/container/garnet is only producing `latest` and a specific version tags. I've opened https://github.com/microsoft/garnet/issues/319 to get semantic version tags, which will allow us to use version `1` or `1.0`, which will allow us to get fixes without taking breaking changes.

Reply:
We have images that are tagged with major.minor.revision today -- how is that different than a this tag...yes it's not semver, but we're not getting the semver benefit of the others either. until/unless they implement a semver tag (which again, would we use the m.m.r and not benefit from semvers anyway), this isn't different than a versioned tag.

Reply:
> until/unless they implement a semver tag (which again, would we use the m.m.r and not benefit from semvers anyway), this isn't different than a versioned tag.

In https://github.com/microsoft/garnet/issues/319 I'm asking for a `1` or `1.0` tag which means when `1.0.6` comes out we would automatically pick it up.

> We have images that are tagged with major.minor.revision today

We should get those fixed as well.

Reply:
@eerhardt Thank you. They have already implemented https://github.com/microsoft/garnet/issues/319. So I set major.minor Tag version